features/support/pages/admin/moderator_action_section.rb

```diff
@@ -0,0 +1,24 @@
+module Admin
+  # A entry in the moderator action logs
+  class ModeratorActionSection < SitePrism::Section
+    element :summary, "*[data-type=summary]"
+    element :moderator, "*[data-type=moderator]"
+    element :created_at, "*[data-type=taken-at]"
+
+    def rendering?(moderator_action)
```

Comment:
I'm thinking `rendering?` is really the wrong word, what do you think about `#showing?` or `#displaying?`